Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -304,6 +304,10 @@ protected def Homeomorph.inv (G : Type*) [TopologicalSpace G] [InvolutiveInv G]
 lemma Homeomorph.coe_inv {G : Type*} [TopologicalSpace G] [InvolutiveInv G] [ContinuousInv G] :
     ⇑(Homeomorph.inv G) = Inv.inv := rfl
 
+@[to_additive]
+theorem nhds_inv (a : G) : 𝓝 a⁻¹ = (𝓝 a)⁻¹ :=
```

Comment:
Do you think this could be a simp lemma, the other way round?

Reply:
IDK. We rarely do computations with `nhds` filters and "pointwise" notation.

Reply:
True, but that feels like an argument to make it a simp lemma in the reverse direction: it makes the pointwise notation disappear easily